services-transport-api/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.scalecube</groupId>
+    <artifactId>scalecube-services-parent</artifactId>
+    <version>2.5.7-SNAPSHOT</version>
+  </parent>
+  <artifactId>services-transport-api</artifactId>
```

Comment:
let's change it to `scalecube-services-transport-api`.
IMHO it's not appropriate module name - classes which hosted simple helpers/utils to work with netty byte buffer.

Reply:
actually it is `scalecube-services-transport-utils`. what do you think?

Reply:
Done in 0a23a9e